tasks/config/babel.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
       '!**/public/**',
       '!**/node_modules/**',
       '!**/bower_components/**',
-      '!**/__tests__/**'
+      '!**/__tests__/**',
+      '!**/vega_vis/**',
```

Comment:
Could you explain why this line is in here?

Reply:
nope, good catch, probably accidental checkin, reverting.